tests/testutils.py

```diff
@@ -61,12 +61,7 @@ def run_without_project_id(func):
 
 
 def new_monkeypatch():
-    try:
-        return pytest.MonkeyPatch()
-    except AttributeError:
-        # Fallback for Python 3.5
-        from _pytest.monkeypatch import MonkeyPatch
-        return MonkeyPatch()
+    return MonkeyPatch()
```

Comment:
Why not just `return pytest.MonkeyPatch()`?

Reply:
Looks like `pytest.MonkeyPatch()` works for `pytest` 6.2 and up. Our CIs use the latest version so I think it would be safe to update this code.

Reply:
Updated!

Reply:
Should we just set pytest version to 6.2 and up in requirements.txt file?

Reply:
Good idea! Updated.